### PR TITLE
New guidelines

### PR DESCRIPTION
**Pull request by 2bndy5**
This adds a link in the readme.rst to the new guidelines.rst file that includes pictures. I hope this is clear enough for everyone. thoughts? HIGH PRIORITY!

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -48,7 +48,7 @@ Hardware & Sensors:
 3. What Do I Need to Know?
 =============================
 
-When contributing to a organization or community, they'll always recommend you review the "contributing guideline" document. No one ever really reads through the guidelines because they are usually long-winded and can get very complicated. The general rule for contributing is to keep things professional. If you read that previous sentence, then you've just read the our contributing guideline.
+When contributing to a organization or community, they'll always recommend you review the "contributing guideline" document. No one ever really reads through the guidelines because they are usually long-winded and can get very complicated. The general rule for contributing is to keep things professional. Read the our `contributing guidelines <https://github.com/DVC-Viking-Robotics/about-us/blob/new-guidelines/Contributing%20Guidelines.rst>`_.
```
Comment (tejashah88) [inline README.rst]: "Read the our" should be "Read our"

File: Contributing Guidelines.rst [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+       ]
+
+   If you are using Visual Studio, the process is a little different. If you're developing in a basic text editor, you can test the webapp repository by using following command in a terminal/command prompt/powershell:
+
+   .. code-block:: shell
+
+       # make sure you are in the webapp repository's root folder
+       python -m webapp.app
+
+   To exit, use either the "shutdown server" option from within the webapp's GUI (from the drop-down menu of the power icon in the top right corner) or you could use ``ctrl+c`` in the terminal/command prompt/powershell (this ``keyboardInterupt`` method is only recommended when something has gone terribly wrong).
```
Comment (tejashah88) [inline Contributing Guidelines.rst]: - "you could use ``ctrl+c``" can be "you can use ``ctrl+c``"
- remove the  ``keyboardInterupt`` part?

**Review by tejashah88 — APPROVED** [review]
Minor grammar issues, but looks good!

File: README.rst (hunk 1 above)
Comment (2bndy5) [reply README.rst]: good catch

File: Contributing Guidelines.rst (hunk 2 above)
Comment (2bndy5) [reply Contributing Guidelines.rst]: could was meant to emphasize that its not the preferred way. They need to know how to kill it with a `keyboardInterupt` if "something has gone terribly wrong." I should change the terribly wrong part to "when the webapp's GUI isn't getting rendered"

**Comment by 2bndy5** [comment]
@tejashah88 I don't know MD that well... Could you add a section in the webapp readme that links to the guidelines?

File: Contributing Guidelines.rst (hunk 2 above)
Comment (tejashah88) [reply Contributing Guidelines.rst]: I'd add something like "DO NOT USE THIS UNLESS YOU REALLY NEED TO" for proper emphasis (bolding helps too).

**Comment by tejashah88** [comment]
You already did that. Seems like the RST method works for markdown as well.

**Comment by 2bndy5** [comment]
nm I was talking about the readme in the webapp repo, but I just realized its the same syntax I'm using in the comments here

**Pull request merged**